### PR TITLE
Billing Dashboard, Invoice, and History

### LINKED ISSUE #1303: Implement billing history

### LINKED ISSUE #2392: Billing Invoice Page

### LINKED ISSUE #2394: Implement billing dashboard page
* Contact info
* Credit cards
* Recent billing activity
* Account balance

### PR DESCRIPTION
**Pull request by na3d**
Closes #2394
Closes #2392 
Closes #1303

### CODE REVIEW

File: src/api/generic/invoices.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  genConfig, ReducerGenerator, genActions, ONE, MANY,
+} from '~/api/internal';
+
+export const config = genConfig({
+  plural: 'invoices',
+  endpoint: id => `/account/invoices/${id}`,
+  supports: [ONE, MANY],
+  subresources: {
+    _items: {
+      singular: 'items',
```
Comment (eatonphil) [inline src/api/generic/invoices.js]: Again this should be plural not singular. Singular is only for non-paginated endpoints.

Comment (na3d) [reply src/api/generic/invoices.js]: It isn't really paginated, they are just throwing that data in there for ? reason.

File: src/billing/layouts/HistoryPage.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+  render() {
+    const { account, invoices } = this.props;
+    const timezone = getStorage('profile/timezone') || 'UTC';
+
+    return (
+      <div>
+        <section>
+          <Card header={<CardHeader title="Billing History" />}>
+            <Table
```
Comment (eatonphil) [inline src/billing/layouts/HistoryPage.js]: I'd recommend you share a component very similar to VolumesList. The only difference between these two pages is that you are limiting the amount of data in one. This duplication is totally unnecessary.

Comment (na3d) [reply src/billing/layouts/HistoryPage.js]: true, good point

File: src/api/ad-hoc/account.js [hunk 3]
```diff
@@ -7,3 +7,13 @@ export function transferPool() {
     dispatch(actions.one({ _transferpool }, 0));
   };
 }
+
+export function makePayment(amount) {
+  return (dispatch) => dispatch(fetch.post('/account/payments', {
+    usd: parseFloat(amount.toFixed(2)),
```
Comment (eatonphil) [inline src/api/ad-hoc/account.js]: parseFloat should be redundant after the toFixed call?

Comment (na3d) [reply src/api/ad-hoc/account.js]: toFixed turns it into a string

Comment (eatonphil) [reply src/api/ad-hoc/account.js]: Oh duh. Then why are you calling it?

**Pull request merged**